app/views/tag/_tagging.html.erb

```diff
@@ -3,7 +3,7 @@
 <p>
   <span id="tags">
     <% @node.tags.each do |tag| %>
```

Comment:
So then here, we could use `@node.community_tags` instead of `@node.tags` -- that'd get us the /usage/ of the tag, which has an author. See https://github.com/publiclab/plots2/blob/master/app/models/drupal_node.rb#L414-L420 for the difference, and see how `drupal_tag` is associated via `drupal_node_community_tag`, which is a join table: https://github.com/publiclab/plots2/blob/master/app/models/drupal_node.rb#L50